Controller/MediaController.php

```diff
@@ -44,27 +45,28 @@ public function getMedia($id)
     /**
      * @throws NotFoundHttpException
      *
-     * @param string $id
-     * @param string $format
+     * @param Request $request
+     * @param string  $id
+     * @param string  $format
      *
      * @return Response
      */
-    public function downloadAction($id, $format = 'reference')
+    public function downloadAction(Request $request, $id, $format = 'reference')
```

Comment:
Is there a way to simulate this signature on 3.x? I am not sure how to trigger this deprecation